libs/dh/metering-point/feature-chargelink/src/components/create.ts

```diff
@@ -0,0 +1,194 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { NonNullableFormBuilder, ReactiveFormsModule, Validators } from '@angular/forms';
+import { Component, computed, effect, inject, signal, ViewEncapsulation } from '@angular/core';
+
+import { TranslocoDirective, TranslocoPipe } from '@jsverse/transloco';
+
+import { VaterStackComponent } from '@energinet/watt/vater';
+import { WattButtonComponent } from '@energinet/watt/button';
+import { WATT_MODAL, WattTypedModal } from '@energinet/watt/modal';
+import { WattTextFieldComponent } from '@energinet/watt/text-field';
+import { WattDatepickerComponent } from '@energinet/watt/datepicker';
+import { WattDropdownComponent, WattDropdownOptions } from '@energinet/watt/dropdown';
+
+import { lazyQuery, query } from '@energinet-datahub/dh/shared/util-apollo';
+import { DhActorStorage } from '@energinet-datahub/dh/shared/feature-authorization';
+
+import {
+  ChargeType,
+  EicFunction,
+  GetChargeByTypeAndOwnerDocument,
+  GetSyoMarketParticipantDocument,
+} from '@energinet-datahub/dh/shared/domain/graphql';
+
+@Component({
+  selector: 'dh-metering-point-create-charge-link',
+  imports: [
+    TranslocoPipe,
+    TranslocoDirective,
+    ReactiveFormsModule,
+
+    WATT_MODAL,
+    WattButtonComponent,
+    WattDropdownComponent,
+    WattTextFieldComponent,
+    WattDatepickerComponent,
+
+    VaterStackComponent,
+  ],
+  encapsulation: ViewEncapsulation.None,
+  styles: `
+    dh-metering-point-create-charge-link {
+      watt-button .mdc-button.mat-mdc-button {
+        width: 100%;
```

Comment:
suggestion: Let's not rely on an implementation detail of `watt-button`. Move this override to `watt-button` instead. I'd argue it is a good default for all button variants, but maybe just limit it to "selection" type buttons for now.